cmake/Abseil.cmake

```diff
@@ -0,0 +1,22 @@
+include(FetchContent)
+
+set(ABSL_ENABLE_INSTALL ON)
```

Comment:
should we use `BENCHMARK_DOWNLOAD_DEPENDENCIES` too from GoogleTest.cmake?

Reply:
I considered this. The way I see it is that GoogleTest is not a required dependency - it's only so if you want tests enabled. Abseil is a required dependency.